qldbdriver/statement_execution_test.go

```diff
@@ -55,9 +55,11 @@ func TestStatementExecutionIntegration(t *testing.T) {
 	testBase := createTestBase()
 	testBase.deleteLedger(t)
 	testBase.createLedger(t)
+	defer testBase.deleteLedger(t)
 
 	qldbDriver, err := testBase.getDriver(ledger, 10, 4)
 	require.NoError(t, err)
+	defer qldbDriver.Shutdown(context.Background())
```

Comment:
Had to refresh my `defer` knowledge! This preserves execution order (i.e. driver shutdown -> delete ledger) because `defer`s are stacked [[docs](https://go.dev/blog/defer-panic-and-recover)] 👍